openhtf/io/output/mfg_inspector.py

```diff
@@ -131,6 +132,7 @@ def _MangleMeasurement(name, value, measurement, mangled_parameters):
       mangled_name += '_'
     mangled_param = test_runs_pb2.TestParameter()
     mangled_param.name = mangled_name
+    mangled_param.associated_attachment = attachment.name
```

Comment:
if you only use the name, might as well just pass `attachment_name` instead of the whole attachment object.